ldk/javascript/examples/self-test-loop/src/tests/screen/index.ts

```diff
@@ -109,3 +111,101 @@ export async function performOcr() {
       });
   });
 }
+
+console.log(`Starting app`);
+
+async function testOCRUsingCursor() {
+  let i = 0;
+  let cursorPositionStream: Cancellable;
+  let topParam: number;
+  let leftParam: number;
+  let topParam1: number;
+  let leftParam1: number;
+  await sleep(1000);
+  cursor
+    .listenPosition((response) => {
```

Comment:
Looks like listenPosition simply polls the cursor position every .5 seconds, that seems like it might be difficult to setup a box around specific text.